sdks/python/apache_beam/examples/complete/juliaset/setup.py

```diff
@@ -31,7 +31,8 @@
 import subprocess
 from distutils.command.build import build as _build
 
-import setuptools
+# workaround pylint bug: https://github.com/PyCQA/pylint/issues/3152
```

Comment:
How about we file an issue and add a comment:
# TODO(BEAM-...): re-enable lint check. 
We can tag the issue as newbie/starter/trivial and it can be someone's first contribution to beam.